src/makeRobot.js

```diff
@@ -38,7 +14,59 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      const nameInfo = `name: ${this.name}`;
+      const versionInfo = `chip version: ${this.version}`;
+      const wheelInfo = `wheels: ${this.wheels}`;
+
+      return `${nameInfo}, ${versionInfo}, ${wheelInfo}`;
+    },
+    get location() {
```

Comment:
Add empty line between methods. Fix it everywhere
```suggestion
    },
    
    get location() {
```